lib/std/posix.zig

```diff
@@ -720,9 +720,9 @@ pub fn raise(sig: u8) RaiseError!void {
     }
 
     if (native_os == .linux) {
+        // block all signals; see https://github.com/ziglang/zig/pull/22337#issuecomment-2771259869
```

Comment:
Would be better to just reproduce my comment here:

```suggestion
        // https://git.musl-libc.org/cgit/musl/commit/?id=0bed7e0acfd34e3fb63ca0e4d99b7592571355a9
        //
        // Unlike musl, libc-less Zig std does not have any internal signals for implementation purposes, so we
        // need to block all signals on the assumption that any of them could potentially fork() in a handler.
```